analyzer/src/main/kotlin/managers/Yarn.kt

```diff
@@ -69,6 +70,8 @@ class Yarn(
 
     override fun getRemotePackageDetails(workingDir: File, packageName: String): JsonNode {
         val process = run(workingDir, "info", "--json", packageName)
-        return jsonMapper.readTree(process.stdout)["data"]
+        return jsonMapper.readTree(process.stdout)["data"] ?: jsonMapper.readTree(process.stderr)["data"].also { yarnErr ->
```

Comment:
But with `yarnErr ->` the line becomes too long and you need to wrap it. So maybe stick to the implicit `it`after all...

Reply:
done